Matlab/Matlab.sublime-syntax

```diff
@@ -62,7 +62,7 @@ contexts:
       scope: comment.double.percentage.matlab
       captures:
         1: punctuation.definition.comment.matlab
-    - match: '%\{'
+    - match: '%\{\s*\n'
```

Comment:
The regex should be `(%\{)\s*\n`, so that the `punctuation` scope is applied to the `%{`.